furusiya/main_interface.py

```diff
@@ -24,6 +25,9 @@ class Game:
     current_turn = None
     playing = False  # True when in-game, false otherwise
 
+    saveload = None
+    keybinder = None
```

Comment:
Should be `key_binder` which is a weird name. Binder (thing that holds papers) with a keys in it?

Reply:
I say this is good as-is. Think keybinds. Keybinder. Makes sense, no?